package.json

```diff
@@ -9,7 +9,7 @@
   "description": "Static site generator for frontend developers",
   "dependencies": {
     "bows": "1.4.8",
-    "jquery": "2.1.3",
+    "jquery": "3.1.1",
```

Comment:
In the meantime jQuery went up to v3.2.0. We should fix that right before finally having green light